book/projects/project_github.md

```diff
@@ -25,26 +20,27 @@ We encourage all teams to create a new GitHub repository within the {{hackweek}}
 
 * Choose "Create a new repository"
 * Name your repository using a short name.
-* We recommend choosing "Public" (so other participants can follow progress). You can also select Private for now and make Public before final presentation.
+* We recommend choosing "Public" so other participants can follow progress
+* Add a "topic" to your repository details (click on the gear icon next to the "About" section on your repository page) to help others find your work (e.g. `uw-hackweek-snowex2024`)
 
 Visit the [GitHub documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) for more details on creating a repository from a template.
 
 ## Create a new GitHub team within the {{hackweek}} Organization
 GitHub teams provide a convenient way for you to set permissions on your project repository for groups of people, rather than having to do this for each individual on your team. 
 
 1. Go to the "Teams" tab near the top of the GitHub {{hackweek}} Organization page: {{ '[https://github.com/orgs/{name}/teams](https://github.com/orgs/{name}/teams)'.format(name=hackweek) }}
-1. Click green “New team” button
-1. Use your short project name for "Team name"
-1. Set the team to visibility "Visible"
-1. Click on the "Create Team" button
-1. Add all team members by clicking the “Members” tab and the green “Add a member” button
+2. Click green “New team” button
+3. Use your short project name for "Team name"
+4. Set the team to visibility "Visible"
+5. Click on the "Create Team" button
+6. Add all team members by clicking the “Members” tab and the green “Add a member” button
 
-### Add the project team to your project repo
+## Add the project team to your project repo
 
 1. Go to the new repository you just created 
-1. Click “Settings” tab in upper right corner
-1. Select “Manage access” on left side of page
-1. Add your team name under the "Manage access" section
-1. Select “write” to enable all team members to make commits
+2. Click “Settings” tab in upper right corner
```

Comment:
In markdown land, I think it is preferred to let markdown take care of the numbering and only use `1.` to indicate a numbered list. Frees one up of having to manage the sequence.

Reply:
wow! today I learned! thanks :)